perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -52,11 +62,23 @@ echo checkout PR branch
 git checkout pr/$KOKORO_GITHUB_PULL_REQUEST_NUMBER
 echo Mounting gcs bucket from pr branch
 mkdir -p gcs
+BUCKET_NAME=presubmit-perf-test
 # The VM will itself exit if the gcsfuse mount fails.
 go run . $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 # Running FIO test
 chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
+sudo umount gcs
+
+# Executing implicitdir integration tests for the testbucket flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --testbucket=gcsfuse-implicitdir-tests
```

Comment:
what is the diff between running against a test bucket vs all tests?

Reply:
for --testbucket, first it will [mount that bucket](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/setup/setup.go#L189) with given flags and then run the tests.

for mounted dir flag, we already mounted gcsfuse in some folder and using it for testing.

as we introduced both the flags, I thought I should test for both.

Reply:
not required. testing using bucket should be covered in GKE tests.